src/utils/indexedDB.ts

```diff
@@ -1,7 +1,33 @@
 const DB_NAME = 'exampleDB';
 const DB_VERSION = 1;
 const STORE_NAME = 'certificates';
-import { Certificate } from '../components/data/data';
+import { Certificate, Supplier } from '../components/data/data';
+
+export const searchSuppliers = async (
```

Comment:
This is what I wrote here and what you need to use on supplier search. So you have this method implemented but you are not using it anywhere